src/library.js

```diff
@@ -596,6 +596,11 @@ LibraryManager.library = {
   atexit__proxy: 'sync',
   atexit__sig: 'ii',
   atexit: function(func, arg) {
+#if ASSERTIONS
+#if NO_EXIT_RUNTIME == 1
+    Runtime.warnOnce('atexit() called, but NO_EXIT_RUNTIME, so atexits() will not be called. set NO_EXIT_RUNTIME to 0 (see the FAQ)');
```

Comment:
This should probably say 'but NO_EXIT_RUNTIME is 1' (or 'is set') instead of just 'but NO_EXIT_RUNTIME'